### PR TITLE
WIP Created team member cards

### PR DESCRIPTION
**Pull request by annsudhart**
The JSON file `src/pages/about-us/members.json ` and `public/boardmembers` folder can be modified to change the divisions displayed on the about page, let me know if there is anything I can do to optimize this!

### CODE REVIEW

File: src/components/boardmember/style.less [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    -ms-flex-direction: column;
+    background-color: #F0F0F0;
+    padding: 10px 0px;
+    margin: 10px;
+    border-radius: 20px;
+    box-shadow: 3px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.board-member-photo {
+    width: @card-width;
```
Comment (leestanley) [inline src/components/boardmember/style.less:15]: Is there a way we can increase the card size and possibly shrink it down using a media query on mobile? This is what the width looks like on the Figma 
image

File: src/components/boardmember/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+const BoardMember: React.FC<BoardMemberProps> = ({ name, title, photo, }) => (
+  <div className="board-member-card">
+      <div className="board-member-photo" style={{backgroundImage: 'url('+photo+')'}} />
+      <div className="board-member-info">
+          <div className="board-member-title">
+            {name}
+          </div>
+          <div className="board-member-description">
```
Comment (leestanley) [inline src/components/boardmember/index.tsx:17]: Should be styled on the Figma the name is larger than the role + a larger font weight as well 
image

File: src/pages/about-us/style.less [hunk 3]
```diff
@@ -33,6 +33,40 @@
...
 
+.board-members {
+  label {
+    display: inline-flex;
+    padding: 1em 0;
+    font-size: 3em;
+    font-weight: bolder;
+  }
+
+  select {
```
Comment (leestanley) [inline src/pages/about-us/style.less:44]: When you select an option theres a blue outline on chrome you can solve it using [this quick css fix](https://stackoverflow.com/questions/31570444/remove-blue-highlighting-of-option)
image

File: src/components/boardmember/style.less (hunk 1 above)
Comment (annsudhart) [reply src/components/boardmember/style.less:15]: Looks like the desktop vs mobile cards have differing padding values so I added that into the CSS

File: src/components/boardmember/index.tsx (hunk 2 above)
Comment (annsudhart) [reply src/components/boardmember/index.tsx:17]: I made the text bolder and changed the fonts slightly for the name and role - I couldn't really see the exact font sizes in the Figma so I did 1.1 em and 0.8em

Comment (leestanley) [reply src/components/boardmember/index.tsx:17]: To increase the size of the name I made it 1.2 rem and added a margin-bottom of 5px 
image

**Comment by leestanley** [comment]
I'm gonna label this WIP and make it a feature branch blocked until we have the full board-list + profile pics

File: src/components/boardmember/index.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  name: string;
+  title: string;
+  photo: string;
+}
+
+const BoardMember: React.FC<BoardMemberProps> = ({ name, title, photo, }) => (
+  <div className="board-member-card">
+      <div className="board-member-photo" style={{backgroundImage: 'url('+photo+')'}} />
+      <div className="board-member-info">
+          <div className="board-member-title">
```
Comment (leestanley) [inline src/components/boardmember/index.tsx:14]: Why wrap it in a div versus an h2/h3?

Comment (annsudhart) [reply src/components/boardmember/index.tsx]: tbh I thought of `h2`/`h3` as only section headers and not really headers for `div`s, it's probably better to use those elements

**Pull request closed without merging**